### PR TITLE
Rename k2_sherpa to k2

### PR DESCRIPTION
**Pull request by xianbaoqian**
Context https://github.com/huggingface/hub-docs/pull/552#discussion_r1048464435

It's called k2_sherpa in the inference API but in the repos, people just use k2 so let's make it consistent so that the export script can work.

### CODE REVIEW

File: .github/workflows/python-api-k2.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # TODO: Merge this with allenNLP to have a single workflow for all docker images.
```
Comment (osanseviero) [inline .github/workflows/python-api-k2.yaml:1]: Would you mind adding a second workflow as this one? https://github.com/huggingface/api-inference-community/blob/main/.github/workflows/python-api-flair-cd.yml 

That will enabled continuous deployment

**Review by osanseviero — COMMENTED** [review]
LGTM but it might require some changes in api-inference-community. 

@Narsil I don't see k2 in places such as https://github.com/huggingface/api-inference/blob/main/master/app/types.py#L21, but the API works. How come? :D

**Comment by xianbaoqian** [comment]
> LGTM but it might require some changes in api-inference-community.
> 
> @Narsil I don't see k2 in places such as https://github.com/huggingface/api-inference/blob/main/master/app/types.py#L21, but the API works. How come? :D

Yeah. I'm curious on this too.

**Comment by Narsil** [comment]
It's defined right there: https://github.com/huggingface/api-inference/blob/main/master-rs/src/data/hf_types.rs#L237

**Comment by Narsil** [comment]
The rename should work.

**Comment by Narsil** [comment]
> The rename should work.

Unfortunately the build is now broken, I don't know what maintainers have done.

**Review by Narsil — APPROVED** [review]
LGTM

**Comment by xianbaoqian** [comment]
Thanks for fixing the bug @Narsil !

**Pull request merged**